t/python/t0013-job-list-inactive.py

```diff
@@ -0,0 +1,263 @@
+        flux.job.submit(self.fh, compute_jobreq, waitable=True)
+
+    attrs = [
+        "userid",
+        "state",
+        "name",
+        "ntasks",
+        "t_submit",
+        "t_run",
+        "t_inactive",
+        ]
+
+    # should return an empty list if there are no inactive jobs
+    def test_00_list_inactive_expect_empty_list(self):
+        rpc_handle = flux.job.job_list_inactive(self.fh, time.time() - 3600, 10, self.attrs)
+
+        try:
+            jobs = rpc_handle.get_jobs()
+        except EnvironmentError as e:
+            print("{}: {}".format("rpc", e.strerror), file=sys.stderr)
+            sys.exit(1)
+
+        self.assertEqual(len(jobs), 0)
+
+    # make sure one job is read from RPC
+    def test_01_list_inactive_success(self):
+        # submit a sleep 0 job
+        self.submitJob()
+
+        time.sleep(1)
+
+        rpc_handle = flux.job.job_list_inactive(self.fh, time.time() - 3600, 10, self.attrs)
+
+        try:
+            jobs = rpc_handle.get_jobs()
+        except EnvironmentError as e:
+            print("{}: {}".format("rpc", e.strerror), file=sys.stderr)
+            sys.exit(1)
+
+        self.assertEqual(len(jobs), 1)
+
+    # multiple jobs submitted should return a longer list of inactive jobs
+    def test_02_list_multiple_inactive(self):
+        mult_jobs = []
+
+        # submit a bundle of sleep 0 jobs
+        for i in range(10):
+            jobid = self.submitJob()
+            mult_jobs.append(jobid)
+
+        for jobid in mult_jobs:
+            job.wait(self.fh)
+
+        rpc_handle = flux.job.job_list_inactive(self.fh, time.time() - 3600, 20, self.attrs)
+
+        try:
+            jobs = rpc_handle.get_jobs()
+        except EnvironmentError as e:
+            print("{}: {}".format("rpc", e.strerror), file=sys.stderr)
+            sys.exit(1)
+
+        self.assertEqual(len(jobs), 9)
+
+    # flux job list-inactive with since = 0.0 should return all inactive jobs
+    def test_03_list_all_inactive(self):
+        rpc_handle = flux.job.job_list_inactive(self.fh, 0.0, 20, self.attrs)
+
+        try:
+            jobs = rpc_handle.get_jobs()
+        except EnvironmentError as e:
+            print("{}: {}".format("rpc", e.strerror), file=sys.stderr)
+            sys.exit(1)
+
+        self.assertEqual(len(jobs), 9)
+
+    # flux job list-inactive with max_entries = 5 should only return a subset
+    def test_04_list_subset_of_inactive(self):
+        rpc_handle = flux.job.job_list_inactive(self.fh, 0.0, 5, self.attrs)
+
+        try:
+            jobs = rpc_handle.get_jobs()
+        except EnvironmentError as e:
+            print("{}: {}".format("rpc", e.strerror), file=sys.stderr)
+            sys.exit(1)
+
+        self.assertEqual(len(jobs), 5)
+
+    # flux job list-inactive with the most recent timestamp should return len(0)
+    def test_05_most_recent_inactive(self):
+        for i in range(5):
+            jobid = self.submitJob()
```

Comment:
should wait on `jobid` to ensure it completes before tests below?  Or perhaps wait could be added to `submitJob()` so that you don't have to call `job.wait()` all the time?   Similar below in other tests.